htdocs/libraries/icms/member/user/Object.php

```diff
@@ -117,7 +117,7 @@ public function __construct(&$handler, $data = array()) {
         $this->initVar('email', self::DTYPE_STRING, null, true, 60, null, null, _US_EMAIL);
         $this->initVar('url', self::DTYPE_STRING, null, false, 255, null, null, _US_WEBSITE);
         $this->initVar('user_avatar', self::DTYPE_FILE, null, false, 30, null, null, _US_AVATAR);
-        $this->initVar('user_regdate', self::DTYPE_INTEGER, date('U'), false, null, null, null,
+        $this->initVar('user_regdate', self::DTYPE_INTEGER, null, false, null, null, null,
```

Comment:
I have updated my previous comment ;) Forgot to write a bit more code there to make it work.